### ISSUE TITLE
Error: node_modules/json-bigint/lib/parse.js:112

### ISSUE THREAD

**Issue by OlivierCuyp** [mailjet/mailjet-apiv3-nodejs#15]
Hi,

I tried  this lib with a simple hardcoded exemple like below.
In the exemple I just replace the sensitive info by tokens like <TOKEN>.
Trying it with node v4.2.4, I get the following error:

```
<PROJECT_PATH>/node_modules/node-mailjet/node_modules/json-bigint/lib/parse.js:112
            throw {
            ^
SyntaxError: Unexpected ''
```

Any idea on this one ?

The code:

```js
var Mailjet = require('node-mailjet').connect('<MJ_APIKEY_PUBLIC>', '<MJ_APIKEY_PRIVATE>');

var emailData = {
  FromEmail: 'server@<MY_DOMAIN>',
  FromName: 'MV Studio',
  Subject: 'Test with the NodeJS Mailjet wrapper',
  'Text-part': 'Hello NodeJs !',
  Recipients: [{ Email: '<MY_EMAIL>' }],
  Attachments: []
};

var sendEmail = Mailjet.post('send');

sendEmail
.request(emailData)
.then(function (response) {
  console.log('Response', response);
})
.catch(function (err) {
  console.error('Error', err);
});
```

**Comment by GuillaumeBadi** [comment]
Hey @OlivierCuyp !
What version of the wrapper are you currently using ? Is it the last one ?

**Comment by GuillaumeBadi** [comment]
I might have spotted the problem here. Your code works perfectly for me, but I got your same error if I provide invalid credentials. Thanks to have spotted this.

Can you print out the API keys before using them to make sure they are valid ?

**Comment by GuillaumeBadi** [comment]
The fix is on master, can you check it works for you before I make the release on NPM? The error should now be properly handled.

Guillaume

**Comment by OlivierCuyp** [comment]
Hi @GuillaumeBadi.

I'll update the package and make the test.
I guess I get a better error message now.
I'll let you know. soon.

**Comment by OlivierCuyp** [comment]
Ok, I just re-installed the master with : `npm i git+ssh://git@github.com:mailjet/mailjet-apiv3-nodejs.git`

And running the same script gives me a much better error :

```
Error { [Error: Unsuccesfull]
  ErrorMessage: 'Unauthorized',
  statusCode: 401,
  response:
   Response {
     domain: null,
     _events: {},
     _eventsCount: 0,
     _maxListeners: undefined,
     res:
      IncomingMessage {
  ...
```

**Comment by OlivierCuyp** [comment]
PS: FYI, I corrected my api key and I got the email.

**Comment by GuillaumeBadi** [comment]
Published on NPM: v3.0.3 :)
